dts/arm/nxp/nxp_rt118x.dtsi

```diff
@@ -90,6 +90,22 @@
 		gpio-controller;
 		#gpio-cells = <2>;
 	};
+
+	/* GPT1 is used for the hardware timer, not as a standard counter */
+	gpt_hw_timer: gpt@46C0000 {
```

Comment:
```suggestion
	gpt_hw_timer: gpt@46c0000 {
		compatible = "nxp,gpt-hw-timer";
		reg = <0x46c0000 0x4000>;
```
style nit- make the alphabetic characters in the register addresses lowercase

Reply:
Updated.